sdk/sdk.go

```diff
@@ -22,80 +19,22 @@ type SDK struct {
 
 	ps *pubsub.PubSub
 
-	m         manager.Manager
 	container container.Container
 	db        database.ServiceDB
 	execDB    database.ExecutionDB
 }
 
 // New creates a new SDK with given options.
-func New(m manager.Manager, c container.Container, db database.ServiceDB, instanceDB database.InstanceDB, execDB database.ExecutionDB) *SDK {
+func New(c container.Container, db database.ServiceDB, instanceDB database.InstanceDB, execDB database.ExecutionDB) *SDK {
 	ps := pubsub.New(0)
 	return &SDK{
-		Service:   servicesdk.New(m, c, db, execDB),
+		Service:   servicesdk.New(c, db, execDB),
 		Instance:  instancesdk.New(c, db, instanceDB),
-		Execution: executionsdk.New(m, ps, db, execDB),
+		Execution: executionsdk.New(ps, db, execDB, instanceDB),
 		Event:     eventsdk.New(ps, db),
```

Comment:
Same as previous comments.
Each SDK should have only access to its specific resource database (eg instance SDK accesses only instance DB, and so on).